deegree-docker/Dockerfile

```diff
@@ -13,6 +13,13 @@ ENV consoleSecretKey=000001544E797221:564344F65B8F9DDBA6A410E461E7801E10955F56D8
 ENV apiUser=deegree
 ENV apiPass=deegree
 
+HEALTHCHECK \
+    --interval=30s \
+    --timeout=30s \
+    --start-period=200s \
```

Comment:
I think we need to increase the start period. If it takes 15 minutes on hale-connect, with some buffer, I would rather use something like `1200s`

Reply:
yes, I can do that.
Unforunatly it was not possible to use ENV variables for the parameter values cause docker rejected to build that. I guess there is a default behaviour to overwrite this from the deployment setup.